cbmc: 5.71.0 -> 5.74.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/cbmc/versions.


meta.description for cbmc is: "CBMC is a Bounded Model Checker for C and C++ programs"

meta.homepage for cbmc is: "http://www.cprover.org/cbmc/"

meta.changelog for cbmc is: ""


###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/diffblue/cbmc/releases/tag/cbmc-5.72.0)

- [Compare changes on GitHub](https://github.com/diffblue/cbmc/compare/cbmc-5.71.0...cbmc-5.72.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 5.72.0 with grep in /nix/store/mwx5jn8nq13sqabnchdi3f78vr9jx3i9-cbmc-5.72.0
- found 5.72.0 in filename of file in /nix/store/mwx5jn8nq13sqabnchdi3f78vr9jx3i9-cbmc-5.72.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
cbmc
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/mwx5jn8nq13sqabnchdi3f78vr9jx3i9-cbmc-5.72.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A cbmc https://github.com/r-ryantm/nixpkgs/archive/1566a36fb5fbbfb14ab9143ccf53a44c589d734c.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/mwx5jn8nq13sqabnchdi3f78vr9jx3i9-cbmc-5.72.0
ls -la /nix/store/mwx5jn8nq13sqabnchdi3f78vr9jx3i9-cbmc-5.72.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

---

###### Maintainer pings

cc @jiegec for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

jiegec commented:
Result of `nixpkgs-review pr 205274` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

jiegec commented:
Result of `nixpkgs-review pr 205274` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

jiegec commented:
Build fails on aarch64-linux: https://logs.nix.ci/?key=nixos/nixpkgs.205274&attempt_id=2ee09057-dc54-4cbc-88c1-8da12631faf8

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/cbmc/versions.


meta.description for cbmc is: "CBMC is a Bounded Model Checker for C and C++ programs"

meta.homepage for cbmc is: "http://www.cprover.org/cbmc/"

meta.changelog for cbmc is: ""


###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/diffblue/cbmc/releases/tag/cbmc-5.72.1)

- [Compare changes on GitHub](https://github.com/diffblue/cbmc/compare/cbmc-5.71.0...cbmc-5.72.1)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 5.72.1 with grep in /nix/store/65hv7z1xlvspghpqxvfy2qamx04qh0q1-cbmc-5.72.1
- found 5.72.1 in filename of file in /nix/store/65hv7z1xlvspghpqxvfy2qamx04qh0q1-cbmc-5.72.1

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
cbmc
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/65hv7z1xlvspghpqxvfy2qamx04qh0q1-cbmc-5.72.1 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A cbmc https://github.com/r-ryantm/nixpkgs/archive/7e640c8c81c67cd89443097d3b178fac345edf80.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/65hv7z1xlvspghpqxvfy2qamx04qh0q1-cbmc-5.72.1
ls -la /nix/store/65hv7z1xlvspghpqxvfy2qamx04qh0q1-cbmc-5.72.1/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

---

###### Maintainer pings

cc @jiegec for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

jiegec commented:
Result of `nixpkgs-review pr 205274` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/diffblue/cbmc/releases.


meta.description for cbmc is: "CBMC is a Bounded Model Checker for C and C++ programs"

meta.homepage for cbmc is: "http://www.cprover.org/cbmc/"

meta.changelog for cbmc is: ""


###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/diffblue/cbmc/releases/tag/cbmc-5.72.2)

- [Compare changes on GitHub](https://github.com/diffblue/cbmc/compare/cbmc-5.71.0...cbmc-5.72.2)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 5.72.2 with grep in /nix/store/qd64hp1as3qf1zjcm1jccapw8fzjpiml-cbmc-5.72.2
- found 5.72.2 in filename of file in /nix/store/qd64hp1as3qf1zjcm1jccapw8fzjpiml-cbmc-5.72.2

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
cbmc
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/qd64hp1as3qf1zjcm1jccapw8fzjpiml-cbmc-5.72.2 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A cbmc https://github.com/r-ryantm/nixpkgs/archive/1ad03f7784df34ae321ac16d4b1c30f63f20bca0.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/qd64hp1as3qf1zjcm1jccapw8fzjpiml-cbmc-5.72.2
ls -la /nix/store/qd64hp1as3qf1zjcm1jccapw8fzjpiml-cbmc-5.72.2/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

---

###### Maintainer pings

cc @jiegec for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

jiegec commented:
Result of `nixpkgs-review pr 205274` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/cbmc/versions.


meta.description for cbmc is: "CBMC is a Bounded Model Checker for C and C++ programs"

meta.homepage for cbmc is: "http://www.cprover.org/cbmc/"

meta.changelog for cbmc is: ""


###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/diffblue/cbmc/releases/tag/cbmc-5.73.0)

- [Compare changes on GitHub](https://github.com/diffblue/cbmc/compare/cbmc-5.71.0...cbmc-5.73.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 5.73.0 with grep in /nix/store/7yb4w28qp09wgd9fry41d45yq1z9rwz8-cbmc-5.73.0
- found 5.73.0 in filename of file in /nix/store/7yb4w28qp09wgd9fry41d45yq1z9rwz8-cbmc-5.73.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
cbmc
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/7yb4w28qp09wgd9fry41d45yq1z9rwz8-cbmc-5.73.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A cbmc https://github.com/r-ryantm/nixpkgs/archive/2062713333fc1421a95f6ebce212b34b7272510e.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/7yb4w28qp09wgd9fry41d45yq1z9rwz8-cbmc-5.73.0
ls -la /nix/store/7yb4w28qp09wgd9fry41d45yq1z9rwz8-cbmc-5.73.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

---

###### Maintainer pings

cc @jiegec for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
cbmc: 5.71.0 -> 5.74.0

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/diffblue/cbmc/releases.


meta.description for cbmc is: CBMC is a Bounded Model Checker for C and C++ programs

meta.homepage for cbmc is: http://www.cprover.org/cbmc/




###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/diffblue/cbmc/releases/tag/cbmc-5.74.0)

- [Compare changes on GitHub](https://github.com/diffblue/cbmc/compare/cbmc-5.71.0...cbmc-5.74.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 5.74.0 with grep in /nix/store/sfd8vdmjjcazmjbkii25xqnmask8xhz8-cbmc-5.74.0
- found 5.74.0 in filename of file in /nix/store/sfd8vdmjjcazmjbkii25xqnmask8xhz8-cbmc-5.74.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

First fifty rebuilds by attrpath
cbmc
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/sfd8vdmjjcazmjbkii25xqnmask8xhz8-cbmc-5.74.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A cbmc https://github.com/r-ryantm/nixpkgs/archive/9aa633ca8c6a86a4628bafd56f0a2925d5591371.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/sfd8vdmjjcazmjbkii25xqnmask8xhz8-cbmc-5.74.0
ls -la /nix/store/sfd8vdmjjcazmjbkii25xqnmask8xhz8-cbmc-5.74.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

---

###### Maintainer pings

cc @jiegec for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

jiegec commented:
Result of `nixpkgs-review pr 205274` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

jiegec commented:
Result of `nixpkgs-review pr 205274` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbmc</li>
  </ul>
</details>

bobby285271 committed:
cbmc: mark as broken on aarch64-linux

Upstream issue has no respond for a month and ofborg keeps reporting this failure.

bobby285271 commented:
@ofborg build cbmc.passthru.tests
